ecs-cli/modules/compose/ecs/utils/convert_task_definition.go

```diff
@@ -48,6 +48,11 @@ var supportedComposeYamlOptions = []string{
 
 var supportedComposeYamlOptionsMap = getSupportedComposeYamlOptionsMap()
 
+type volumes struct {
+	volumeHasHost   map[string]string
```

Comment:
`volumeHasHost` seems like something that you'd use for a bool. Can you use a better name here?